include/emp/prefab/README.md

```diff
@@ -0,0 +1,112 @@
+# Prefab Components
+
+The prefabricated components in Empirical compose basic components and add
+features to make more complicated user interfaces. Here's a log of some
+stylistic and design choices that might be useful in future development.
+
+### Overriding Streaming
+
+Most components use the stream operator (`<<`) to append other components
+inside. However, many prefabricated components have multiple layers of divs and
```

Comment:
Can you say it more descriptively/precisely than “append other components inside” (maybe something like “add user-defined content“)